### ISSUE TITLE
Add support for multiple clusters / versions

### ISSUE THREAD

**Issue by PhilippeAB** [saltstack-formulas/postgres-formula#174]
Need the port to be configurable

**Comment by myii** [comment]
@PhilippeAB Port is now configurable.

**Comment by myii** [comment]
@noelmcloughlin I don't think this issue should be closed just yet.  Allowing the port to be configurable was just the first stage.  As I've said in https://github.com/saltstack-formulas/postgres-formula/pull/218#issuecomment-397809521, this progress can now be built upon to support multiple clusters/versions.

**Comment by noelmcloughlin** [comment]
True. I closed because the issue description only requested a configurable port. I now see slogan is talking about multiple cluster / version. Thats fine.
